readme_template.md

```diff
@@ -8,7 +8,7 @@
 [![license](https://img.shields.io/github/license/StevenBlack/hosts.svg)](https://github.com/StevenBlack/hosts/blob/master/license.txt)
 [![repo size](https://img.shields.io/github/repo-size/StevenBlack/hosts.svg)](https://github.com/StevenBlack/hosts)
 [![contributors](https://img.shields.io/github/contributors/StevenBlack/hosts.svg)](https://github.com/StevenBlack/hosts/graphs/contributors)
-[![Build Status](https://travis-ci.org/StevenBlack/hosts.svg?branch=master)](https://travis-ci.org/StevenBlack/hosts)
+[![Build Status](https://img.shields.io/travis/StevenBlack/hosts/master.svg)](https://travis-ci.org/StevenBlack/hosts)
```

Comment:
This is unnecessary. The URL was already correct.

Reply:
I never said it wasn't correct. This is consistent with all other badges and also faster (not that it matters, though).

Reply:
There is a difference between using the URL from the one who provides us the service and from a third party for that one. 

Don't understand me wrong, I know it's consistent but using the Travis one make sense in that case.

But it's my opinion.